src/i18n/strings/en_EN.json

```diff
@@ -1618,5 +1618,9 @@
     "If you didn't remove the recovery method, an attacker may be trying to access your account. Change your account password and set a new recovery method immediately in Settings.": "If you didn't remove the recovery method, an attacker may be trying to access your account. Change your account password and set a new recovery method immediately in Settings.",
     "Failed to set direct chat tag": "Failed to set direct chat tag",
     "Failed to remove tag %(tagName)s from room": "Failed to remove tag %(tagName)s from room",
-    "Failed to add tag %(tagName)s to room": "Failed to add tag %(tagName)s to room"
+    "Failed to add tag %(tagName)s to room": "Failed to add tag %(tagName)s to room",
+    "Sounds": "Sounds",
+    "Custom Notification Sounds": "Notification sound",
+    "Set a new custom sound": "Set a new custom sound",
+    "Reset to default sound": "Reset to default sound"
```

Comment:
Please use the i18n script so these get sorted correctly into the file rather than manually inserting them